src/data/sidenav.yml

```diff
@@ -60,6 +60,8 @@
           permalink: /language/mixins
         - title: Enums
           permalink: /language/enums
+        - title: Dot Shorthands
+          permalink: /language/dot-shorthand
```

Comment:
Opinion on making this plural? I think I always refer to this feature as dot shorthands.